jobs/garden/spec

```diff
@@ -147,6 +147,10 @@ properties:
     description: "Hard limit for the tcp buffer memory in bytes. When set to zero tcp memory limit is disabled."
     default: 0
 
+  garden.disable_swap_limit:
+    description: "If true, container is not going to be limited in swap space. Useful when swap space is not configured on the VM."
```

Comment:
let's make this a bit more explicit: "Should only be used if swap is disabled on the VM" or similar? (we can do this when merging, don't need to bother to update PR just for this)